server/game/cards/12-KotI/HeirToTheIronThrone.js

```diff
@@ -18,11 +18,16 @@ class HeirToTheIronThrone extends PlotCard {
 
     cardSelected(player, card) {
         this.game.addMessage('{0} uses {1} to search their deck and put {2} into play', player, this, card);
+        let dupeCard = player.getDuplicateInPlay(card);
         player.putIntoPlay(card);
-
-        this.game.queueSimpleStep(() => {
-            this.promptForSacrifice(player);
-        });
+        //only prompt for sacrifice if the card put into play wasn´t a duplicate
+        if(!dupeCard) {
+            this.game.queueSimpleStep(() => {
+                this.promptForSacrifice(player);
+            });
+        } else {
+            this.game.addMessage('{0} can not sacrifice a character because {1} is put into play as a duplicate', player, card);
```

Comment:
I don't think we should explain rules via game messages. If we add this, we'd also have to go back to all `Then` abilities and add similar things (e.g. you save with a dupe pre-Then for Mag the Mighty, etc).

Reply:
good point, it is explained on the corresponding thronesdb page and i guess there rule explanations belong.
fixed